tests/test_scooby.py

```diff
@@ -96,17 +96,17 @@ def test_ipy():
 
 def test_get_version():
     name, version = scooby.get_version(numpy)
-    assert version == numpy.__version__
+    # assert version == numpy.__version__
     assert name == "numpy"
 
-    # Package that was no version given by owner; gets 0.1.0 from setup/pip
+    # Package that has no `__version__` but has `0.1.0` from distribution
     name, version = scooby.get_version("no_version")
     assert version == "0.1.0"
     assert name == "no_version"
 
-    # Dummy module without version (not installed properly)
+    # Path dummy module (not installed properly)
     name, version = scooby.get_version("dummy_module")
-    assert version == scooby.report.VERSION_NOT_FOUND
+    assert version == scooby.report.MODULE_NOT_FOUND
```

Comment:
IMO, this is desired behavior, though, a breaking change...

If we continue to support this, then I have to bring back all the logic to manually import modules and check for common version attributes which I'd really like to step away from and use `importlib.metadata` instead.

IMO, it is a dangerous and improper scenario for users to install packages by adding them to their path. I'd want `scooby` to pick up on that scenario to alert users/whoever is receiving the `Report` ... perhaps a compromise would be to still fallback to manually importing the package to see if it's in the path when `importlib.metadata.version` fails and give a warning message (but still not check for version attributes)?

Reply:
Would it work with a module in the path that has a version? I think a simple `try import module` to check if it _can_ be imported would be enough, not much more trickery. If it is `MODULE_NOT_FOUND`, but can be imported, we could simply print "not installed"? As a compromise? Otherwise "not found".

Reply:
> Would it work with a module in the path that has a version?

Just tried locally and nope.

> I think a simple try import module to check if it can be imported would be enough, not much more trickery.

That's fair... and perhaps just a check for `__version__` and if not fail? I want to keep the logic as simple as possible

> but can be imported, we could simply print "not installed"? As a compromise? Otherwise "not found".

I'd like to maybe put `(not installed)` next to the version if it imports successfully (next to the version if it has it)

Reply:
> I'd like to maybe put (not installed) next to the version if it imports successfully (next to the version if it has it)

Yeah, that sounds cool.

Reply:
Might work from Python 3.11 onwards? https://github.com/python/cpython/issues/94113

Reply:
Well, how about that... I'll add some code to handle this with a note to remove it when we drop 3.10 support (woah... the future is upon us)

Reply:
Okay, this is resolved now. @prisae, please review